magda-web-client/src/homepage-config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * This is only a sample file.
+ * Should be hosted at either:
+ * https://s3-ap-southeast-2.amazonaws.com/data-gov-au-frontpage/development/ (for development)
+ * or https://s3-ap-southeast-2.amazonaws.com/data-gov-au-frontpage/production/ (for production)
+ * backgroundImageUrl must contain image width as part of url /name. e.g. twenty-1440w.jpg (image width is 1440px)
+ */
+window.magda_client_homepage_config = {
+    baseUrl:
+        "https://s3-ap-southeast-2.amazonaws.com/data-gov-au-frontpage/development/",
+    backgroundImageUrls: [
+        "twenty-1440w.jpg",
+        "twenty-720w.jpg",
+        "twenty-0w.jpg",
+        "twenty-1080w.jpg",
+        "twenty-2160w.jpg",
+        "twenty-2880w.jpg"
+    ],
+    tagLineTextDesktop: "An easy way to find, access and reuse public data",
+    tagLineTextMobile: "Find, share, and reuse open government data",
+    Lozenge: {
+        icon: "camara.svg",
+        text: "Explore QLD road data",
+        url: "https://search.data.gov.au/search?q=QLD%20road%20data"
```

Comment:
"Queensland road data" will work way better than "QLD road data"

Reply:
I think the text is the tiny button on the homepage. 
![image](https://user-images.githubusercontent.com/674387/37940847-b5e74db6-31b6-11e8-94f3-73a9bf2b991a.png)
I guess use abbreviation is mainly for control its size --- correct me if I am wrong @tkeuneman 
This config file is included in repo only for reminding us its structure in future.
The actual copy is located at: https://s3-ap-southeast-2.amazonaws.com/data-gov-au-frontpage/development/homepage-config.js
(staging & production site will load file in `staging` & `production` folder).
Most changes to homepage later can be done via modifying this config on s3.

Reply:
We can leave the text as "QLD", just change the text in the URL to "queensland" :).

I don't mind if it's in S3 or here.

Reply:
Oh I see. I thought you want to change the button text.
Will change the URL now.